server/models/migrations/next/migrate.sql

```diff
@@ -0,0 +1,36 @@
+/* migrate v1.21.0 to next */
+
+/**
+author: @jniles
+date: 2021-08-30
+description: adds cost columns and indices to relevant tables
+*/
+
+DROP TABLE IF EXISTS `cost_center_aggregate`;
+CREATE TABLE `cost_center_aggregate` (
+  `period_id`       MEDIUMINT(8) UNSIGNED NOT NULL,
+  `debit`           DECIMAL(19,4) UNSIGNED NOT NULL DEFAULT 0.00,
+  `credit`          DECIMAL(19,4) UNSIGNED NOT NULL DEFAULT 0.00,
+  `cost_center_id`  MEDIUMINT(8) UNSIGNED NOT NULL,
+  `principal_center_id` MEDIUMINT(8) UNSIGNED NULL,
+  KEY `cost_center_id` (`cost_center_id`),
+  KEY `principal_center_id` (`principal_center_id`),
+  KEY `period_id` (`period_id`),
+  CONSTRAINT `cost_center_aggregate__period` FOREIGN KEY (`period_id`) REFERENCES `period` (`id`),
+  CONSTRAINT `cost_center_aggregate__cost_center_id` FOREIGN KEY (`cost_center_id`) REFERENCES `fee_center` (`id`),
+  CONSTRAINT `cost_center_aggregate__principal_center_id` FOREIGN KEY (`principal_center_id`) REFERENCES `fee_center` (`id`)
+) ENGINE=InnoDB DEFAULT CHARACTER SET = utf8mb4 DEFAULT COLLATE = utf8mb4_unicode_ci;
+
+
+ALTER TABLE `posting_journal` ADD COLUMN `cost_center_id`      MEDIUMINT(8) UNSIGNED NULL;
```

Comment:
I suggest you use the add_column_if_missing() function to add these new columns

Reply:
Whoops, good point.